packages/gatsby-theme-docs/src/loaders/code-sample.js

```diff
@@ -0,0 +1,8 @@
+function codeSample(content) {
+  this.cacheable();
+
+  this.value = content;
+  return `module.exports = ${JSON.stringify(content)}`;
```

Comment:
So basically you're wrapping the file content into an export.

What type is the `content`? A string or what?